shell/shared/renderSession/RenderSession.h

```diff
@@ -45,6 +49,10 @@ class RenderSession {
   void setCurrentQuadLayer(size_t layer) noexcept {
     currentQuadLayer_ = layer;
   }
+    
+  [[nodiscard]] igl::TextureFormat getDepthTextureFormat() const {
```

Comment:
What's the reason for this addition? The point of suggested+requested configs is to let a particular session choose the formats it wants without needing to create some kind of dummy session without a platform.

Reply:
You want the stencil buffer to be disabled by default. So, I can only move the depthTextureFormat field from RenderSessionConfig to RenderSession, and then NanovgRenderSession can override the default depthTextureFormat.

Reply:
Anything already merged into `main` is fine. For any further changes, aim to make only the minimal changes necessary. Please update your diffs to minimize the modifications required to add NanoVG.

Reply:
The minimal way to make the changes is to keep the depthTextureFormat field in RenderSessionConfig. 

If I want to show the NanoVG session and make changes to depthTextureFormat locally, and avoid committing it to the main branch. 

I can add comments in NanovgSession to explain that other developers can make such changes if they want to show the effect.

This approach is also well-suited for Android because without it, the changes required for Android would be significant as well.

 What do you think about this approach?